AnnoyancesFilter/sections/popups.txt

```diff
@@ -2893,9 +2893,9 @@ naturalnews.com#$#body { overflow: visible !important; }
 onitsukatiger.com##.osr-overlay
 python-3.ru##body > div[class="modal"]
 bartinolay.com###siteAcilis8Saniye
-usak.tv#$#.modal-backdrop { display: none!important; }
-usak.tv#$#body { overflow: visible!important; }
-usak.tv#$##modal-ads { display: none!important; }
+truyentranhaudio.online,usak.tv#$#.modal-backdrop { display: none!important; }
```

Comment:
As I see, this is anti-adblock. But I cannot reproduce.
If it can be closed(or no), better move to anti-adblock related file.